CHANGELOG.md

```diff
@@ -15,7 +15,14 @@ Increment the:
 
 ## [Unreleased]
 
-* Added `ForceFlush` to `TracerProvider` in SDK. ([#588](https://github.com/open-telemetry/opentelemetry-cpp/pull/588)).
+## [0.2.0] 2021-03-02
+
+* [SDK] Added `ForceFlush` to `TracerProvider` in SDK. ([#588](https://github.com/open-telemetry/opentelemetry-cpp/pull/588)).
+* [SDK] Added Resource API
```

Comment:
Consider putting the actual PR link (similar like what we have for #588).

Reply:
Good point. Done now.